performance/src/main/java/org/hibernate/validator/performance/simple/SimpleValidation.java

```diff
@@ -67,10 +68,11 @@
 	@Threads(50)
 	@Warmup(iterations = 10)
 	@Measurement(iterations = 50)
-	public void testSimpleBeanValidation(ValidationState state) {
+	public void testSimpleBeanValidation(ValidationState state, Blackhole bh) {
 		DriverSetup driverSetup = new DriverSetup( state );
 		Set<ConstraintViolation<Driver>> violations = state.validator.validate( driverSetup.getDriver() );
 		assertThat( violations ).hasSize( driverSetup.getExpectedViolationCount() );
+		bh.consume( violations );
```

Comment:
In practice, the effect might be the same. However, the purpose of Blackhole is to prevent Dead-code elimination, False sharing and Write wall, which can skew benchmark results.  I am not sure what effect .hasSize() has on the aforementioned JIT optimisations.